.gitignore

```diff
@@ -23,3 +23,4 @@ _testmain.go
 *.test
 *.prof
 *.bin
+.vscode
```

Comment:
Probably should be `.vscode/` to ensure it only matches dir. Super-ultra-pedantic-nit-pick here.